trainer/finetune.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2020 Google LLC
```

Comment:
Comments on other files. __init__.py is empty? Is it still needed? Also, do we need to check in the __pycache__ stuff?

Reply:
The init files are needed for python to recognize the folder a module. The pycache is where python stores bytecode stuff but it's auto generated when you run the code so I don't think we need to check it in.

Reply:
Remove from commit and add a .gitignore for __pycache__?

Reply:
yeah I removed the pycache files